Shopify/cli
# Fix websockets not working on node <17

### WHY are these changes introduced?
We detected that our reverse proxy was not working with websockets if the node version was 16 or lower.

Our reverse-proxy solution was based on fastify http proxy, after long debugging sessions I couldn't find the issue. Websocket requests were not properly redirected.

Because of this issue, dev-console or extensions live-reload weren't working properly.

Fixes https://github.com/Shopify/cli/issues/157 



### WHAT is this pull request doing?
- Replace `fastify/http-proxy` with [node-http-proxy](https://github.com/http-party/node-http-proxy)



### How to test your changes?
Update `.nvmrc` and `dev.yml` to use node version 16.15.1
`dev up`
`dev shopify app dev --path fixtures/app`
Open the dev-console and it should load all extensions.
